doc/topics/releases/carbon.rst

```diff
@@ -17,3 +17,24 @@ Grains Changes
   .. code-block:: jinja
 
       {% set on_vmware = grains['virtual'].lower() == 'vmware' %}
+
+Deprecations
+============
+- The ``boto_vpc`` execution module had two functions removed,
```

Comment:
With these, let's provide the full name. `boto_vpc.associate_new_dhcp_options_to_vpc` versus the way it's currently phrased. That will make it easier to read and understand, I think.